bin/compile

```diff
@@ -71,13 +71,22 @@ else
   minecraft_url="https://s3.amazonaws.com/Minecraft.Download/versions/${minecraft_version}/minecraft_server.${minecraft_version}.jar"
 fi
 
-echo -n "-----> Installing Minecraft ${minecraft_version}... "
+if [[ -z "${MINECRAFT_SERVER_URL}" ]]; then
+  echo "No custom server URL set..."
```

Comment:
i think you could combine this if/else with the previous if/else because setting a custom URL would take precedence over setting a custom version

Reply:
I'd remove the message about not using a custom URL too.